src/Sylius/Behat/Context/Api/Shop/CartContext.php

```diff
@@ -30,6 +30,9 @@ final class CartContext implements Context
     /** @var ApiClientInterface */
     private $cartsClient;
 
+    /** @var ApiClientInterface */
+    private $productClient;
```

Comment:
```suggestion
    private $productsClient;
```
to be consistent with the service above